### ISSUE TITLE
FEATURE REQUEST: Please change tray icon to NextCloud logo

### ISSUE THREAD

**Issue by bcurran3** [nextcloud/desktop#2131]
### Expected behaviour

NextCloud Client's tray icon would best represent the program if it used NextCloud's actual program logo. 

![image](https://user-images.githubusercontent.com/14026600/85908331-7eb01580-b7c9-11ea-8fed-09756598c858.png)

### Actual behaviour

The current green circle with a check is not intuitive at all to recognize it's purpose. I often look at it and think to myself "What the heck is this icon?" as it's, IMHO, not very useful in it's current state: if https://github.com/nextcloud/desktop/issues/1184 was implemented, the tray icon would be useful on a daily basis instead of rare and random basis.

![image](https://user-images.githubusercontent.com/14026600/85908166-e1ed7800-b7c8-11ea-9015-abccd0d89dbc.png)

### Steps to reproduce

Run NextCloud client on Windows.

### Client configuration

Client version: 2.6.4stable-Win64 (build 20200303)

Operating system: Windows 10 2004

OS language: English

Qt version used by client package (Linux only, see also Settings dialog): n/a

Client package (From Nextcloud or distro) (Linux only): n/a

Installation path of client: C:\Program Files\Nextcloud

### Server configuration

n/a

### Logs

n/a

**Comment by er-vin** [comment]
@jancborchardt opinion there?

**Comment by jancborchardt** [comment]
You are welcome to read the whole background discussion about why specifically we are _not_ using the Nextcloud icon for the "all synced" state at https://github.com/nextcloud/desktop/pull/416 :)

The short answer is – to quote [myself from that thread](https://github.com/nextcloud/desktop/pull/416#issuecomment-397246488):

> The logo as default state will be confusing as every other system uses a definite checkmark for this state. I'd really prefer we keep the Nextcloud icon for the "not set up yet"/"connecting" state since that is the one you will see initially, and because a logo is not proper feedback that everything is fine.

**Comment by bcurran3** [comment]
(I had a feeling that was going to be the reasoning and I'm prepared with counterpoint.)

@jancborchardt  A check-mark superimposed over the NextCloud logo would both identify the program and show state. 

Better yet, have the logo shown all the time, and have it blinking or spinning during sync.

Whatever method is chosen to show state, industry standard is to show the icon. 

Using the program icon = instant recognition as it matches the graphic imprinted on the user's brain via the website, the installation, the icon they click to run the program, the icon when the program is displayed, etc. etc. 

Case in point:

![image](https://user-images.githubusercontent.com/14026600/86381511-4766bc00-bc42-11ea-83ff-290252b4870b.png)
![image](https://user-images.githubusercontent.com/14026600/86384426-8d704f80-bc43-11ea-99da-98aba4074560.png)

Simply put, it makes it intuitive.

I hope you'll reconsider.

**Comment by MegaV0lt** [comment]
I see that in the same way.
Using the current icon leafes everyone back with an icon that gives no clu what program it is.

Imagine if everyone is doing the same as nextcloud. All icons in the tray will look the same! Awful

**Comment by stavros-k** [comment]
I'll have to agree also with @bcurran3. I find my self a lot of times trying to find NC icon, I'm used to rely on notification bar too much. I have 3 screens with 1 of them having NC root folder open constantly but i still look on the tray to see if everything is fine. (I also have 3 accounts connected. (admin, personal, work))
